packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```
@@ -2311,8 +2309,10 @@ test('mutable liquidity triggers and interest sensitivity', async t => {
     manualTimer.tick();
   }
   await waitForPromisesToSettle();
-  aliceUpdate = await E(aliceNotifier).getUpdateSince();
-  bobUpdate = await E(bobNotifier).getUpdateSince();
-  t.is(aliceUpdate.value.vaultState, Phase.ACTIVE);
+  bobUpdate = await E(bobNotifier).getUpdateSince(bobUpdate.updateCount);
   t.is(bobUpdate.value.vaultState, Phase.LIQUIDATED);
+
+  // No change for Alice
+  aliceUpdate = await E(aliceNotifier).getUpdateSince(); // can't use updateCount because there's no newer update
```

`updateCount` is optional. You don't need to justify not including it. Not including it means "I want the current state, even if it hasn't changed."

---

I had to include it in the other places or the update would come back before the liquidation processor settled. There are more promise resolutions now with "liquidate one at a time".

I actually think we should make a habit of including the `updateCount` so that you have a guarantee that you're getting a new state. If you don't expect a new state then don't ask for it and if you do then the test should fail if it's not available.

I even think it might be worth a helper wrapper around this notifier/response pair so you could have,
```
const aliceAfterLiquidation = await alicesUpdates.next();
```
(it would do the `updateCount` stuff itself to ensure you're getting a new update)